database/src/database.py

```diff
@@ -1,29 +1,44 @@
-""" TODO: 
-        - error handling for commands
 """ 
+    TODO:
+        - better handle clearing of collections so it doesn't run everytume connect() is called
+"""
 import pymongo
 import sys
 
 DB_NAME = 'mongodb'
 DB_PORT = 27017
 HOST = 'localhost'
 
+ACCOUNTS_COLLECT = "accounts"
+TRANSACT_COLLECT = "transactions"
+TRIGGER_COLLECT = "triggers"
+
+
 # Code created with reference to
 # https://medium.com/analytics-vidhya/setting-up-a-database-class-mongodb-for-interactions-in-python-6a16417cd58e
 
 class Database(object):
+
     DATABASE = None
 
     def connect():
+        """
+            Connect to database and clear current collections
+        """
+
         try:
             client = pymongo.MongoClient(HOST, DB_PORT)
             Database.DATABASE = client[DB_NAME]
             print("-----CONNECTED TO MONGODB DB-----")
-        
+            Database.DATABASE[ACCOUNTS_COLLECT].drop()
+            Database.DATABASE[TRANSACT_COLLECT].drop()
+            Database.DATABASE[TRIGGER_COLLECT].drop()
```

Comment:
Just something to watch out here is that each time a webserver is started the database will be cleared.

Which is probably fine.